src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    public static final int MAX_TASKS = 100;
+    public static String horizontal = ("____________________________________________________________\n");
```

Comment:
```suggestion
    public static String HORIZONTAL= ("____________________________________________________________\n");
```

Perhaps you can rename this with all caps, since this is a String constant